### DISCUSSION TITLE
Location of the Device List and Config?

### DISCUSSION THREAD

**Discussion by Stonemage** [airsquared/blobsaver#467] [Q&A] [answered]
I had to rebuild my Mac for various reasons, and I can't boot back into the old drive - I am just getting kernel panics.

Where does Blobsaver save the original device list on the disk, and is there a way to simply copy/paste the original device list and config files to the new install?

I have already moved my /Users/<username>/Blobs folder... but I want to be able to copy the device list without having to read from every device again.

Thanks!

**Comment by airsquared** [comment, marked as answer]
Sorry for the late response, but please refer to [this wiki](https://github.com/airsquared/blobsaver/wiki/Where-is-blobsaver's-data-stored%3F) page.
